ArJpctBaseLib/src/main/java/org/artoolkit/ar/base/markers/MarkerConfig.java

```diff
@@ -0,0 +1,16 @@
+package org.artoolkit.ar.base.markers;
+
+/**
+ * Represents an ARToolkit marker configuration string.
+ *
+ * @see https://artoolkit.org/documentation/doku.php?id=4_Android:android_developing#loading_markers
+ */
+public interface MarkerConfig
+{
+    /**
+     * Get the ARToolkit native marker string.
+     *
+     * @return the ARToolkit native marker string.
+     */
+    String get();
```

Comment:
get is too generic as a name, I think it should be getConfigString, getFormattedString, getArToolKitString or even just getString.